package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "web-team-tv",
+  "version": "1.0.0",
+  "description": "This is the package for the web team TV",
+  "main": "webpack.config.js",
+  "scripts": {
+    "build": "webpack --progress",
+    "lint": "eslint src/**/*.{js,jsx} *.js; exit 0",
+    "release": "webpack --progress --optimize-minimize",
+    "serve": "webpack-dev-server --port $PORT",
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "watch": "webpack --watch"
+  },
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/hyperoslo/web-team-tv.git"
+  },
+  "author": "",
+  "license": "ISC",
+  "bugs": {
+    "url": "https://github.com/hyperoslo/web-team-tv/issues"
+  },
+  "homepage": "https://github.com/hyperoslo/web-team-tv#readme",
+  "dependencies": {
+    "classnames": "^2.2.0",
+    "react": "0.14.2",
+    "react-addons-css-transition-group": "0.14.2",
+    "react-dom": "0.14.2"
+  },
+  "devDependencies": {
+    "babel-core": "^5.8.23",
+    "babel-eslint": "^4.1.1",
+    "babel-loader": "^5.3.2",
+    "css-loader": "^0.17.0",
+    "eslint": "^1.5.1",
+    "eslint-config-airbnb": "^0.1.0",
+    "eslint-config-hyperoslo": "^1.0.0",
+    "eslint-loader": "^1.0.0",
+    "eslint-plugin-react": "^3.4.2",
```

Comment:
This is a pretty old version, isn't it? The `react/sort-comp` rule that you've disabled in your ESLint config has also been fixed in https://github.com/yannickcr/eslint-plugin-react/pull/429, which should be present in the latest version. Any reason you're not updating?